src/Illuminate/Database/Query/Builder.php

```diff
@@ -1299,6 +1299,28 @@ public function orWhereYear($column, $operator, $value = null)
         return $this->addDateBasedWhere('Year', $column, $operator, $value, 'or');
     }
 
+    /**
+     * Add an "where today" statement in the query.
+     *
+     * @param  string $column
+     * @return \Illuminate\Database\Query\Builder|static
+     */
+    public function whereToday($column)
+    {
+        return $this->whereDate($column, '=', today());
```

Comment:
You can't use `today()`, it's a global helper from Laravel Framework itself.

However `illuminate/database` is a self-contained code base/package which can be used outside of Laravel, thus you can't depend on this.

Also:  it misses passing the time zone, this is vital. Not everyone would require to only use the applications default timezone.

Reply:
I have changed Carbon::today() to DateTime with additional $timeZone parameter